modules/rpc/handler.go

```diff
@@ -38,31 +39,49 @@ type contextInboundMiddleware struct {
 	service.Host
 }
 
-func (f contextInboundMiddleware) Handle(ctx context.Context, req *transport.Request, resw transport.ResponseWriter, handler transport.UnaryHandler) error {
+func (f contextInboundMiddleware) Handle(
+	ctx context.Context,
+	req *transport.Request,
+	resw transport.ResponseWriter,
+	handler transport.UnaryHandler,
+) error {
 	stopwatch := stats.RPCHandleTimer.
 		Tagged(map[string]string{stats.TagProcedure: req.Procedure}).
 		Timer(req.Procedure).
 		Start()
 	defer stopwatch.Stop()
 
-	ctx = ulog.ContextWithLogger(ctx, ulog.Logger(ctx))
+	if span := opentracing.SpanFromContext(ctx); span != nil {
```

Comment:
Ah, thought this was a well known thing. Added a comment in case it throws off people.